django/db/migrations/state.py

```diff
@@ -448,16 +448,17 @@ def reconstruct_manager(mgr):
                 instance = manager_class(*args, **kwargs)
             # We rely on the ordering of the creation_counter of the original
             # instance
-            managers[mgr.name] = (mgr.creation_counter, instance)
+            name = force_text(mgr.name, strings_only=True)
```

Comment:
Yeah, makes sense. I just dropped the flag. There shouldn't be a way `manager.name` ever being `None` in this context.